packages/debian-lib/changelog

```diff
@@ -14,7 +14,7 @@ gdrcopy (@FULL_VERSION@) stable; urgency=low
 
  -- Pak Markthub <pmarkthub@nvidia.com>  Mon, 01 Feb 2021 11:59:59 -0700
 
-gdrcopy (2.1) stable; urgency=low
```

Comment:
Does this work? Can you attribute the changelog to the gdrcopy metapackage instead?
just a question, not a blocker

Reply:
It works. But I decide to also include changelog and readme for the meta package. On the other hand, changelog for libgdrapi and gdrcopy-tests packages has been set to initial version -- those packages were parts of gdrcopy package but this is the first time they exist on their own.

Can I ask you to re-review?